components/plots/plot-types/apparent.js

```diff
@@ -61,7 +61,7 @@ export class ApparentLightCurveOptions extends LightCurveOptions {
   formatDetections(detections, band) {
     return detections
       .filter(function (x) {
-        return x.fid === band && x.corrected
+        return x.fid === band && x.corrected && x.magpsf_corr < 100
```

Comment:
This will make some points dissapear on the apparent plot, don't know if its the desired behaviour of this plot.
(@fforster)

Reply:
@cvalenzu  @fforster  An example of file to download.
[ZTF18abdkamb_20200805.zip](https://github.com/alercebroker/component-library/files/5031016/ZTF18abdkamb_20200805.zip)